src/Utilities/FlowAnalysis/FlowAnalysis/Analysis/PointsToAnalysis/PointsToAnalysisData.cs

```diff
@@ -49,6 +59,8 @@ private PointsToAnalysisData(PointsToAnalysisData data1, PointsToAnalysisData da
             data1.AssertValidPointsToAnalysisData();
             data2.AssertValidPointsToAnalysisData();
             AssertValidPointsToAnalysisData();
```

Comment:
This assert uses `_isDisposable` I think, but we set `_isDisposable` after the assert is done. Should it be moved earlier? Are we missing a test case where the assert should fail with the current code? or does assigning before the assert not matter at all?